### PR TITLE
feat(components/list): pagination icons customization

### PR DESCRIPTION
**Pull request by sdwvit**
add ability to specify custom icons

**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
Icons are hardcoded to be font-awesome

**What is the new behavior?**
Icons can be specified by handy api

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

File: output/components.eslint.txt [hunk 1]
```diff
@@ -18,6 +18,12 @@ Scoping to packages that match 'react-talend-components'
   2:1  error  '@kadira/react-storybook-addon-info' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies
   7:2  error  Unexpected require()                                                                                      global-require
 
+/home/travis/build/Talend/ui/packages/components/src/List/Toolbar/Pagination/Pagination.component.js
```
Comment (talend-sromain) [inline output/components.eslint.txt]: Please fix lints ;)

Comment (sdwvit) [inline output/components.eslint.txt]: done

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://387.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://387.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://387.talend.surge.sh)

**Pull request merged**